### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by fgsoftware1** [fgsoftware1/fgengine#3] [Ideas]
Let your suggestions here

**Comment by fgsoftware1** [comment]
probably embed mono for c# scripting

**Comment by charlieduzstuf** [comment]
Lua/Luau/Lua5.1 support 

( https://github.com/fgsoftware1/fgengine/discussions/4 )

**Reply by charlieduzstuf** [reply]
https://github.com/aap/librw and maybe this

**Reply by fgsoftware1** [reply]
> Lua/Luau/Lua5.1 support

my engine don't support directx

**Reply by charlieduzstuf** [reply]
Lua hasn't got to have directx, does it

**Reply by fgsoftware1** [reply]
> Lua hasn't got to have directx, does it

it runs whatever people make it to run. librw have

**Reply by charlieduzstuf** [reply]
Why was there a problem if u don't have to use directx and u can make it run with other stuff

**Reply by fgsoftware1** [reply]
> Why was there a problem if u don't have to use directx and u can make it run with other stuff

cause this is a lib of other engine

**Reply by charlieduzstuf** [reply]
Ohhh, I see, any way around that
